doc/nrf/known_issues.rst

```diff
@@ -1468,7 +1512,19 @@ Subsystems
 Bluetooth® LE
 =============
 
```

Comment:
Please add the following new issue (we haven't got a workaround yet):
.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0

NCSDK-19942: HID samples don't work with Android 13
  The peripheral_hids_keyboard and peripheral_hids_mouse enter a connection-disconnection loop when trying to connect to them from Android 13.